main.browser.js

```diff
@@ -0,0 +1,7 @@
+require('babel-polyfill');
+import {Client4} from './client/client4.js';
+import {WebSocketClient} from './client/websocket_client.js';
+const client = new Client4();
+
+window.MattermostClient = client;
```

Comment:
We should just build this as a library instead of loading the clients onto the window. Not sure why this file exists. You would just import the clients into your project.